controllers/front/CmsController.php

```diff
@@ -31,10 +31,21 @@ class CmsControllerCore extends FrontController
     /** @var string */
     public $php_self = 'cms';
     public $assignCase;
+
+    /**
+     * @deprecated Since 8.1, it will be protected on next major.
```

Comment:
could you mention the related getters which should be used now in every deprecation?
like 
```suggestion
     * @deprecated Since 8.1, it will become protected in next major version. Use getCms() method instead.
```
or maybe with a `@see tag`
```
 * @deprecated Since 8.1, it will become protected in next major version.
 * 
 * @see getCms()
```

both cases ok for me, as long as the alternative is mentioned

Reply:
@zuk3975 Fixed ✅